src/internal/types/themeData.ts

```diff
@@ -0,0 +1,15 @@
+export type ThemeSaveState = {
+  history: any; // json object
+  index: number;
+  hash: string;
+};
+
+export type ThemeHistories = {
+  histories: History[];
+  index?: number;
```

Comment:
I don't think we want this optional either. Is there a case where this wouldn't have a value?